src/crypto/hkdf.ts

```diff
@@ -0,0 +1,22 @@
+import hmac from 'js-crypto-hmac'
+import { logger } from 'utils/logger'
+
+/**
+ * HKDF HMAC-SHA256
+ *
+ * @param key hmac key
+ * @param salt salt
+ * @returns a 32 bytes value being the output HMAC
+ */
+export async function hmacSha256 (key: Uint8Array, salt: Uint8Array): Promise<Uint8Array> {
```

Comment:
Are we really using a JS implementation on HKDF?

Reply:
deleting